pkg/skaffold/generate_pipeline/generate_pipeline.go

```diff
@@ -36,7 +36,14 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func Yaml(out io.Writer, config *latest.SkaffoldConfig, profile *latest.Profile) (*bytes.Buffer, error) {
+// Struct keeps track of config files and their corresponding SkaffoldConfigs and generated Profiles
```

Comment:
The comment should start with the struct name
```suggestion
// ConfigFile keeps track of config files and their corresponding SkaffoldConfigs and generated Profiles
```